src/MSBuild.UnitTests/CommandLineSwitches_Tests.cs

```diff
@@ -731,6 +731,31 @@ public void PreprocessSwitchIdentificationTests()
             Assert.True(unquoteParameters);
         }
 
+        [Fact]
+        public void TargetsSwitchIdentificationTests()
+        {
+            CommandLineSwitches.ParameterizedSwitch parameterizedSwitch;
+            string duplicateSwitchErrorMessage;
```

Comment:
nit:
You can inline these variable declarations such as `out string duplicateSwitchErrorMessage` below

Reply:
committed